main.tf

```diff
@@ -84,7 +84,6 @@ resource "azurerm_mssql_failover_group" "this" {
 resource "azurerm_mssql_server_extended_auditing_policy" "this" {
   server_id              = azurerm_mssql_server.this.id
   log_monitoring_enabled = true
```

Comment:
Good suggestion! Should we add setting the arguments to `null` when `log_monitoring_enabled` = `true`, or is it good enough to add this comment in the script?

Reply:
One approach is to hard-code the arguments to `null` in `main.tf`, and a second approach is to create variables for the arguments with default value `null`. 

I've tested the first approach and tests pass withous issue, but would like to hear your opinion on; if we instead should go for variables, in the scenario where the user preferres sending the logs to a storage account instead of log analytics?

Reply:
I'm leaning towards the first approach. Sure, variables are nice for certain features that can be toggled on or off, however there are certain things that I think we should enforce, e.g. logs are to be stored in Log Analytics instead of Storage.

Reply:
Agreed 👍🏼